cmd/notation/sign.go

```diff
@@ -55,15 +56,18 @@ Example - Sign a container image using the image digest
 			return runSign(cmd, opts)
 		},
 	}
+	opts.LoggingFlagOpts.ApplyFlags(command.Flags())
 	opts.SignerFlagOpts.ApplyFlags(command.Flags())
 	opts.SecureFlagOpts.ApplyFlags(command.Flags())
 	cmd.SetPflagExpiry(command.Flags(), &opts.expiry)
 	cmd.SetPflagPluginConfig(command.Flags(), &opts.pluginConfig)
-
 	return command
 }
 
 func runSign(command *cobra.Command, cmdOpts *signOpts) error {
+	// set log level
+	ctx, _ := cmdOpts.LoggingFlagOpts.SetLoggerLevel()
```

Comment:
Based on the previous comment, this could be changed to:
```suggestion
	ctx, _ := cmdOpts.LoggingFlagOpts.SetLoggerLevel(command.Context())
```

Reply:
Thanks! updated.